ci-operator/config/openshift-eng/rebasebot/openshift-eng-rebasebot-main.yaml

```diff
@@ -595,8 +595,8 @@ tests:
       commands: |
         UPSTREAM_VERSION=$(curl --silent --header "X-GitHub-Api-Version:2022-11-28" "https://api.github.com/repos/kubernetes/autoscaler/releases" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' | grep -E 'cluster-autoscaler-[0-9.]+' | sort -V | tail -n1)
         rebasebot --source https://github.com/kubernetes/autoscaler:"${UPSTREAM_VERSION}" \
```

Comment:
Should this be `--source-repo`?

Reply:
Autoscaler is not compatible with `_BUILTIN_/source-ref-hooks/latest-stable-release.sh`.  Because that repo has multiple projects. It will require its own custom --source-ref-hook. I won't be changing that in this PR.